plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -3318,7 +3318,12 @@ public void testOptimize()
 
     private List<String> getActiveFiles(String tableName)
     {
-        return computeActual(format("SELECT file_path FROM \"%s$files\"", tableName)).getOnlyColumn()
+        return getActiveFiles(tableName, getQueryRunner().getDefaultSession());
+    }
+
+    private List<String> getActiveFiles(String tableName, Session session)
```

Comment:
`getActiveFiles` result doesn't depend on the session; the fact it uses `SELECT` should be viewed as impl detail, it could go directly to storage, or use Iceberg APIs directly instead

no need to pass session arg here.

Reply:
hmm ok